### DISCUSSION TITLE
Consider renaming "touchpad" → "trackpad" in user-facing strings

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/settings-mouse-touchpad#164] [Ideas]
I feel like both used to be in heavy usage, but over time "trackpad" is definitely the one I hear most commonly used. We've even let it slip in blog posts and social media.

**Comment by JoseExposito** [comment]
Personally, I like touchpad, I feel like trackpad is more of a macOS thing.

In addition, libinput uses touchpad:
https://wayland.freedesktop.org/libinput/doc/latest/touchpads.html

Plus touchpad is more popular in Google Trends as well:

![Screenshot from 2021-02-03 19 49 02](https://user-images.githubusercontent.com/1335948/106794731-871ed780-6659-11eb-895a-9e7cacf1f18b.png)

So, my vote goes to touchpad :smile:

**Reply by cassidyjames** [reply]
I think I just realized why I prefer trackpad: "multi-touch touchpad" feels so clunky :joy:
